docs/FAQ.rst

```diff
@@ -341,4 +343,347 @@ If you are behind a corporate proxy, make sure you have ``proxy type`` and
 This file may be overwritten if you reinstall Intel SGX SDK.
 
 
+Glossary
+========
+
+AES-GCM 256
+    Avalon's authenticated encryption algorithm, with a 96b IV
+    and 128b tag. Encrypts data within a work order request or response.
+    Also used to encrypt a request digest and custom data encryption keys
+
+application type ID
```

Comment:
s/application/Application/
Can we capital character for first letter(kind of title case)

Reply:
Done.